framework/replay_buffer.py

```diff
@@ -96,12 +104,12 @@ def sample_trajectories(
         :return: two lists of mb_size many trajectories
         """
 
-        indices = np.random.choice(self.pos, 2 * mb_size, replace=False)
+        indices = np.random.choice(self.pos - 32, 2 * mb_size, replace=False)
 
         trajectories = [
             self.get_trajectory(
                 start,
-                (start + traj_len) % self.pos,
```

Comment:
This allowed us to generate trajectories where start > end

Reply:
`trajectory_indices = np.arange(start_idx, end_idx)` Output: [] 
`get_trajectory()` returns empty tensors